### ISSUE TITLE
Not able to move robot or mass

### ISSUE THREAD

**Issue by lmulhall-phet** [phetsims/gravity-force-lab#198]
<b>Test Device</b>

MacBook Air

<b>Operating System</b>

macOS 10.14.6

<b>Browser</b>

Chrome 78.0.3904.87

<b>Problem Description</b>

For https://github.com/phetsims/QA/issues/455.

I was messing around in the "Data: JSON" wrapper and I noticed that I wasn't able to move the robot or the mass. I had devtools open and I was using the "device mode" feature. The bug persists if you close devtools.

I'm not sure how to reproduce this issue in the standalone PhET-iO sim, and I'm not sure if it happens in other wrappers.

<b>Visuals</b>

![issue2](https://user-images.githubusercontent.com/31285590/68626432-989a2380-0498-11ea-9372-e81a9a9f8c08.gif)

![Screen Shot 2019-11-11 at 3 37 29 PM](https://user-images.githubusercontent.com/31285590/68626724-38f04800-0499-11ea-9516-48b75d266542.png)

**Comment by zepumph** [comment]
Thanks for the bug report. 

>I'm not sure how to reproduce this issue in the standalone PhET-iO sim

Technically, the data json page is the "standalone" sim, in that it isn't embedded in another wrapper. That link just adds a query parameter to print events to the console. This is only to say that it is strange that this occurs here but not in the standalone version. The `Data: colorized` is quite similar to the json page. Can you reproduce this there?

* Can you get this to reproduce consistently on the JSON page? 
* Can you get this to reproduce without having the "device mode" selection?
* I can't reproduce this immediately on my windows 10 chrome. Are there any other devices you can do this with?
* Extra credit: here is a similar link, but with out data stream logging, added scenery input logging, and on the debug version. If you can reproduce, and want to sent the console output over feel free. https://phet-dev.colorado.edu/html/gravity-force-lab/2.2.0-dev.16/phet-io/gravity-force-lab_all_phet-io_debug.html?postMessageOnError&phetioStandalone&sceneryLog=InputEvent

**Comment by zepumph** [comment]
@lmulhall-phet, today @jessegreenberg and I were not able to reproduce this issue on his mac os. We loaded the data:json page and successfully dragged the sliders in a variety of device modes, all with dev tools open. Please reassign me if you find a way to reproduce. Thanks!

**Comment by zepumph** [comment]
@lmulhall-phet it has a been over a month, and we are moving towards publication, have you had any luck with this?

**Comment by lmulhall-phet** [comment]
@zepumph, I haven’t seen this again. I’m guessing it was a fluke. Feel free to press on.

**Comment by zepumph** [comment]
Thanks! I'm going to close this.

**Comment by lmulhall-phet** [comment]
For https://github.com/phetsims/QA/issues/465. I believe I found another instance of this issue, but not in the same sim:

![issue](https://user-images.githubusercontent.com/31285590/72024842-ae775b00-3233-11ea-9807-318a6ecf7a80.gif)

1. I'm not able to move either the blue dot or the orange dot along the number line.
2. The blue dot and the orange dot are on the same number.

Here's a screenshot of the whole screen:

![Screen Shot 2020-01-08 at 4 28 43 PM](https://user-images.githubusercontent.com/31285590/72024954-0746f380-3234-11ea-8eff-0993cbe563ad.png)

Pretty much the exact same setup as in the original issue with the exception of query parameters. As with the original issue, there doesn't seem to be a surefire way to trigger this. It seems random.

@zepumph, @jbphet since we're seeing the same issue in two different sims, should we move this issue to a common code repo? If so, which one?

**Comment by zepumph** [comment]
I would think that perhaps scenery since it doesn't look like that knob in number line is sun/scenery-phet, and the puller is a custom Node too.

**Comment by jbphet** [comment]
I agree that scenery would be the appropriate repo, but unless there is a clear and repeatable way to recreate the situation, it's unlikely that much will be done on it.  @lmulhall-phet - if you have the time, finding a way to repeat this would greatly increase the chances of it being addressed.

**Comment by lmulhall-phet** [comment]
> finding a way to repeat this would greatly increase the chances of it being addressed.

I will of course try, but this will be a backburner task due to the strange circumstances in which the bug occurs. Moving to scenery.
